docs/contributing/contributing-guide.md

```diff
@@ -0,0 +1,106 @@
+# Contributing guide
+
+All contributions, bug reports, bug fixes, documentation improvements, and enhancements are welcome.
+
+All contributors and maintainers to this project should abide by the [Contributor Code of Conduct](/docs/contributing/code-of-conduct.md).
+
+Learn more about the contributors' roles in the [Roles](/docs/contributing/contributor-roles.md) page.
+
+This document describes how to contribute to the Great Expectations Airflow Provider, covering:
+
+- Overview of how to contribute
+- How to set up the local development environment
+- Running tests
+- Authoring the documentation
+
+## Overview of how to contribute
+
+To contribute to the Great Expectations Airflow Provider project:
+
+1. Create a [GitHub Issue](https://github.com/astronomer/airflow-provider-great-expectations/issues) describing a bug, enhancement, or feature request.
+2. Open a branch off of the `main` branch and create a Pull Request into the `main` branch from your feature branch.
+3. Link your issue to the pull request.
+4. After you complete development on your feature branch, request a review. A maintainer will merge your PR after all reviewers approve it.
+
+
+## Set up a local development environment
+
+Setting up a local development environment involves fulfilling requirements, cloning the repo, and setting up a virtual environment.
+
+### Requirements
+
+- [Git](https://git-scm.com/)
+- [Python](https://www.python.org/) version 3.9 to 3.12
+- [Great Expectations](https://docs.greatexpectations.io/docs/core/set_up_a_gx_environment/install_gx) version 1.3.3+
+- [Apache Airflow®](https://airflow.apache.org/) version 2.1.0+ 
+
+### Clone the repo
+
+```bash
+git clone https://github.com/astronomer/airflow-provider-great-expectations.git
+```
+
+### Set up a virtual environment
+
+You can use any virtual environment tool. The following example uses the `venv` tool included in the Python standard library.
+
+1. Create the virtual environment.
+   ```bash
+   $ python -m venv --prompt my-af-gx-venv .venv
```

Comment:
```suggestion
   $ python -m venv .venv
```

Reply:
With the removal of the environment name `my-af-gx-venv` here, should there be a corresponding update in step 3 below which is currently


> 3. Install the package and testing dependencies.
   > ```bash
   > (my-af-gx-venv) $ pip install -e '.[tests]'
   > ```
   
   Should `(my-af-gx-venv)` be replaced with a different environment name? Should the parenthetical be removed completely?

Reply:
short answer is yes, it should be replaced with `.venv`.

the original command is incorrect and does not result in that output -- the environment name is the final part of the command, `.venv`, so in the original the environment name should have been `.venv` as well.